bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -74,6 +76,10 @@ export default class Whiteboard extends React.Component {
               />
             </g>
           </svg>
+          <SlideControlsContainer
```

Comment:
I would inject this component at the same level as PollingContainer, it could help you fix the bug with 90% height.